eng/scripts/Language-Settings.ps1

```diff
@@ -130,16 +130,21 @@ function Get-python-PackageInfoFromPackageFile ($pkg, $workingDirectory)
 # Stage and Upload Docs to blob Storage
 function Publish-python-GithubIODocs ($DocLocation, $PublicArtifactLocation)
 {
-  $PublishedDocs = Get-ChildItem "$DocLocation" | Where-Object -FilterScript {$_.Name.EndsWith(".zip")}
```

Comment:
@sima-zhu here are the changes accounting for the new artifact type.

I have compared outputs. They line up.